.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+### :tophat: What is the goal?
+
```

Comment:
We could added more sections as:

## Aditional information
<!-- Introduce links or documentation about the change  -->

## Types of changes

<!-- What types of changes does your code introduce? -->
<!-- Put an `x` in all the boxes that apply: -->

- [ ] Docs change / refactoring / dependency upgrade.
- [ ] Bug fix.
- [ ] New feature.
- [ ] Improvement.